lib/utils.js

```diff
@@ -35,17 +36,26 @@ exports.basicAuth = function(username, password) {
 
 exports.findAvailablePort = function(app){
 
-  var port = (process.env.PORT || 3000);
+  var port = null;
+
+  try {
+    port = Number(fs.readFileSync(__dirname+'/../port.tmp'));
```

Comment:
It means the file is hidden by default on Macs. So if users have the folder open it isn't visible and the files don't jump around.